crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -778,7 +767,7 @@ impl<F: Clone, Ctx: PaymentMethodRetrieve>
         let payment_intent_fut = tokio::spawn(
             async move {
                 m_db.update_payment_intent(
-                    m_payment_data_payment_intent,
+                    m_payment_data_payment_intent.clone(),
```

Comment:
you can avoid this clone, redeclare the variable above, this way only expiry has to be cloned